integration/helpers/vite.ts

```diff
@@ -81,21 +96,42 @@ export const viteConfig = {
     `;
     return text;
   },
+  build: ({ assetsInlineLimit, assetsDir }: ViteConfigBuildArgs = {}) => {
+    return dedent`
+      build: {
+        // Detect rolldown-vite. This should ideally use "rolldownVersion"
+        // but that's not exported. Once that's available, this
+        // check should be updated to use it.
```

Comment:
@markdalgleish Oh, I forgot to export it. It should be now available with 6.3.0-beta.7 🙏